x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/api_key_configuration.tsx

```diff
@@ -95,7 +94,11 @@ export const ApiKeyConfig: React.FC<{ hasApiKey: boolean; indexName: string }> =
             'xpack.enterpriseSearch.content.indices.configurationConnector.apiKey.description',
             {
               defaultMessage:
-                'Keep your Elasticsearch API key somewhere safe while you configure your connector.',
+                'First, generate an Elasticsearch API key. This {apiKeyName} key will enable read and write permissions for the connector to index to the created {indexName} index. Save the key in a safe place, as you will need it to configure your connector.',
```

Comment:
"index" as a very and noun in this sentence is a little hard to parse.

Reply:
I'll add documents, but I'm not a copy-editor and tbh don't want to be